src/java/org/apache/cassandra/index/sai/IndexContext.java

```diff
@@ -219,6 +225,16 @@ public IndexContext(@Nonnull String keyspace,
         logger.debug(logMessage("Initialized index context with index writer config: {}"), indexWriterConfig);
     }
 
+    public IndexMetadata getConfig()
+    {
+        return config;
+    }
+
+    public void setConfig(IndexMetadata config)
```

Comment:
What's the purpose of this? It's unlikely to work for any `IndexMetadata` that is passed.

Reply:
Leftover. Should be removed.